### ISSUE TITLE
Can't edit users

### ISSUE THREAD

**Issue by navenedrob** [acquia/headless-lightning#102]
Steps to reproduce:

1. Create new user
2. Try to edit the user

Expected result:

User is able to be edited

```
The website encountered an unexpected error. Please try again later.
TypeError: Argument 1 passed to Doctrine\Common\Inflector\Inflector::camelize() must be of the type string, null given, called in PATH_TO_REPO/docroot/profiles/contrib/headless_lightning/modules/headless_ui/src/Redirect.php on line 17 in Doctrine\Common\Inflector\Inflector::camelize() (line 281 of PATH_TO_REPO/vendor/doctrine/inflector/lib/Doctrine/Common/Inflector/Inflector.php).
Doctrine\Common\Inflector\Inflector::camelize(NULL) (Line: 17)
Drupal\headless_ui\Redirect::entityForm(Array, Object) (Line: 73)
headless_ui_form_user_form_alter(Array, Object, 'user_form') (Line: 501)
Drupal\Core\Extension\ModuleHandler->alter('form', Array, Object, 'user_form') (Line: 830)
Drupal\Core\Form\FormBuilder->prepareForm('user_form', Array, Object) (Line: 272)
Drupal\Core\Form\FormBuilder->buildForm('user_form', Object) (Line: 74)
Drupal\Core\Controller\FormController->getContentResult(Object, Object)
call_user_func_array(Array, Array) (Line: 123)
Drupal\Core\EventSubscriber\EarlyRenderingControllerWrapperSubscriber->Drupal\Core\EventSubscriber\{closure}() (Line: 582)
Drupal\Core\Render\Renderer->executeInRenderContext(Object, Object) (Line: 124)
Drupal\Core\EventSubscriber\EarlyRenderingControllerWrapperSubscriber->wrapControllerExecutionInRenderContext(Array, Array) (Line: 97)
Drupal\Core\EventSubscriber\EarlyRenderingControllerWrapperSubscriber->Drupal\Core\EventSubscriber\{closure}()
call_user_func_array(Object, Array) (Line: 153)
Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object, 1) (Line: 68)
Symfony\Component\HttpKernel\HttpKernel->handle(Object, 1, 1) (Line: 67)
Drupal\simple_oauth\HttpMiddleware\BasicAuthSwap->handle(Object, 1, 1) (Line: 57)
Drupal\Core\StackMiddleware\Session->handle(Object, 1, 1) (Line: 47)
Drupal\Core\StackMiddleware\KernelPreHandle->handle(Object, 1, 1) (Line: 99)
Drupal\page_cache\StackMiddleware\PageCache->pass(Object, 1, 1) (Line: 78)
Drupal\page_cache\StackMiddleware\PageCache->handle(Object, 1, 1) (Line: 40)
Drupal\jsonapi\StackMiddleware\FormatSetter->handle(Object, 1, 1) (Line: 49)
Asm89\Stack\Cors->handle(Object, 1, 1) (Line: 47)
Drupal\Core\StackMiddleware\ReverseProxyMiddleware->handle(Object, 1, 1) (Line: 50)
Drupal\Core\StackMiddleware\NegotiationMiddleware->handle(Object, 1, 1) (Line: 23)
Stack\StackedHttpKernel->handle(Object, 1, 1) (Line: 664)
Drupal\Core\DrupalKernel->handle(Object) (Line: 19)
```

**Comment by navenedrob** [comment]
@bappa-ww I closed these issues because I felt that the team was probably aware of them. Also, they have been fixed in the Lightning distribution. This distribution is using an outdated version of that distribution as a base. By updating, eventually, they should naturally fix the issues I opened.

The Lightning distribution comes with all the modules necessary to create a decouple architecture. For now, it's my opinion that people just use that until this one gets updates.
